docs/core/tools/local-tools-how-to-use.md

```diff
@@ -0,0 +1,191 @@
+
+## Install botsay as a local tool
+
+Install the tool from the package that you created in the preceding tutorial:
+
+```dotnetcli
+dotnet tool install --add-source ./nupkg botsay
+```
+
+This command adds the tool to the manifest file that you created in the preceding step. The command output shows which manifest file the newly installed tool is in:
+
+ ```console
+ You can invoke the tool from this directory using the following command:
+ 'dotnet tool run botsay' or 'dotnet botsay'
+ Tool 'botsay' (version '1.0.0') was successfully installed.
+ Entry is added to the manifest file /Users/name/tools/sub/dotnet-tools.json
+ ```
+
+The *.config/dotnet-tools.json* file now has one tool:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "botsay": {
+      "version": "1.0.0",
+      "commands": [
+        "botsay"
+      ]
+    }
+  }
+}
+```
+
+## Use the tool
+
+Invoke the tool by running the `dotnet tool run` command:
+
+```dotnetcli
+dotnet tool run botsay hello from the bot
+```
+
+## Restore a local tool installed by others
+
+You typically install a local tool in the root directory of the repository. After you check in the manifest file to the repository, other developers can get the latest manifest file. To install all of the tools listed in the manifest file, they can run a single `dotnet tool restore` command.
+
+Open the *.config/dotnet-tools.json* file, and replace the contents with the following JSON:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "botsay": {
+      "version": "1.0.0",
+      "commands": [
+        "botsay"
+      ]
+    },
+    "dotnetsay": {
+      "version": "2.1.3",
+      "commands": [
+        "dotnetsay"
+      ]
+    }
+  }
+}
+```
+
+Making this change is the same as getting the latest version from the repository after someone else installed the package `dotnetsay` for the project directory. 
+
+Run the `dotnet tool restore` command.
+
+```dotnetcli
+dotnet tool restore
+```
+
+The command produces output like the following example:
+
+```console
+Tool 'botsay' (version '1.0.0') was restored. Available commands: botsay
+Tool 'dotnetsay' (version '2.1.3') was restored. Available commands: dotnetsay
+Restore was successful.
+```
+
+Verify that the tools are available:
+
+```dotnetcli
+dotnet tool list
+```
+
+The output is a list of packages and commands, similar to the following example:
+
+```console
+Package Id      Version      Commands       Manifest
+------------------------------------------------------------------------------------------------------
+botsay          1.0.0        botsay         C:\Users\username\Documents\botsay\.config\dotnet-tools.json
```

Comment:
Wouldn't the user have to be in the botsay directory or a subdirectory to be able to invoke the dotnetsay tool locally if that's where the manifest file lives?

Reply:
Yes that's true.